packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -0,0 +1,75 @@
+import { getDeployedProxiedContract } from '@celo/protocol/lib/web3-utils'
+import fs = require('fs')
+import {
+  MultiSigContract,
+  RegistryInstance,
+  ReleaseGoldContract,
+  ReleaseGoldProxyContract,
+} from 'types'
+
+module.exports = async (callback: (error?: any) => number) => {
+  try {
+    const argv = require('minimist')(process.argv.slice(2), {
+      string: ['network', 'grants'],
+    })
+    const registry = await getDeployedProxiedContract<RegistryInstance>('Registry', artifacts)
+    const Multisig: MultiSigContract = artifacts.require('MultiSig')
+    const ReleaseGold: ReleaseGoldContract = artifacts.require('ReleaseGold')
+    const ReleaseGoldProxy: ReleaseGoldProxyContract = artifacts.require('ReleaseGoldProxy')
+    const releases = []
+    const handleJSONFile = async (err, data) => {
+      if (err) {
+        throw err
+      }
+      data = JSON.parse(data)
+      const entity = data.entity
+      const releaseGoldConfigs = data.grants
+      for (const releaseGoldConfig of releaseGoldConfigs) {
+        const releaseStartTime = new Date(releaseGoldConfig.releaseStartTime)
```

Comment:
For reviewer: any suggestions to avoid manually pulling each element out of the struct? TS does not seem to allow passing an interface/dict in for a list of parameters.
I've tried using `.apply()` but that doesn't seem to work.

Reply:
I assume you mean object destructuring? http://budiirawan.com/typescript-destructuring-object/